BioArchLinux/plink1.9/lilac.yaml

```diff
@@ -0,0 +1,11 @@
+build_prefix: extra-x86_64
+maintainers:
+  - github: adityaruplaha
+    email: me@adityaruplaha.me
+post_build_script: |
+  git_pkgbuild_commit()
+update_on:
+  - source: github
+    github: chrchang/plink-ng
+    use_latest_release: true
```

Comment:
use_latest_release option will retrieve latest release tag v2.0.0-a.6.29
 from git repo. however the sources array in the PKGBUILD is using git as source without any tag attached.

Reply:
If you want 1.9.x, you can use [`use_max_tag`](https://nvchecker.readthedocs.io/en/latest/usage.html#check-github:~:text=use%5Fmax%5Ftag,-Set) with [`include_regex`](https://nvchecker.readthedocs.io/en/latest/usage.html#list-options:~:text=include%5Fregex).

Reply:
@lilydjwg Thanks, I'll give that a shot.